package/lib/compileFunctions.test.js

```diff
@@ -767,4 +767,129 @@ describe('CompileFunctions', () => {
       });
     });
   });
+
```

Comment:
It would be great to add test case that covers support for `provider`-level setting as well as how `provider`-level setting is overriden by more specific `function`-level configuration